tt

```diff
@@ -57,6 +55,10 @@ main() {
 		esac
 	fi
 
+	TT_PROJECTS=${TT_PROJECTS:-~/tmp}
+	TT_GIT_BASE=${TT_GIT_BASE:-git@github.com:meet-tarantino}
+	apply_workspace_vars # this may override the above defaults
```

Comment:
interesting, yeah i remember that now, lets cross that bridge when we get to plugins :)